### DISCUSSION TITLE
How can I make CSV file from BIN file

### DISCUSSION THREAD

**Discussion by Daranporn** [jrkerns/pylinac#462] [Q&A]
I have MLC logfile in bin file and I want to analyze it by log_analyzer but I don't know how can I do it ?

**Comment by jrkerns** [comment]
Is there something wrong with the section labeled as such in the docs? https://pylinac.readthedocs.io/en/latest/log_analyzer.html#converting-trajectory-logs-to-csv

**Reply by Daranporn** [reply]
thank you, I can solve this problem :)
